test/TokenTestUtils.js

```diff
@@ -318,12 +342,12 @@ async function getActualState(token) {
             'symbol': symbol,
             'currency': currency,
             'decimals': decimals,
-            'masterMinter': masterMinter,
-            'pauser': pauser,
-            'blacklister': blacklister,
-            'tokenOwner': tokenOwner,
-            'proxiedTokenAddress': proxiedTokenAddress,
-            'proxyOwner': proxyOwner,
+            'masterMinter': upperCaseAddress(masterMinter),
```

Comment:
Aren't these already uppercased when you changed the hardcoded address to be in uppercase? (Also, from what I can tell it looks like the BN conversion function will make them uppercase anyway) Do you think we need these upperCaseAddress calls here?